persistent_player_data_version_231.pdef

```diff
@@ -1469,7 +1469,10 @@ $STRUCT_START eFDPostGamePlayer
 	string{22}	xuid
 	int 		awardId
 	float 		awardValue
-	int 		suitIndex
+	// this used to be "int suitIndex" but that might cause issues with pdiffs that add more suits
+	pilotSuit 	suitIndex
+	// this is just here to avoid changing the size of the pdef without just appending data
+	bool 		unused[3]
```

Comment:
I'm not actually 100% sure if it should be this way around or the unused stuff first?